deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     categories: OpenShift Optional
     certified: "false"
     containerImage: quay.io/kubevirt/hyperconverged-cluster-operator:1.1.0
-    createdAt: "2020-03-09 14:34:54"
+    createdAt: "2020-03-12 16:41:25"
```

Comment:
Can you drop this change?
It will cause a merge conflict when doing the cherry-pick.

Reply:
We are not going to cherry pick this to release-2.3.
But yes, we have to figure out if we can set any special git filter to ignore merge conflicts on the timestamp.

Reply:
Maybe this: https://git-scm.com/docs/gitattributes#_defining_a_custom_merge_driver